Better examples for calling JS methods (#1845)
vaadin/docs: articles/create-ui/element-api/client-server-rpc.asciidoc

[source,java]
----
public void complete() {
    getElement().executeJs("MyModule.complete($0)", true);
}
----

> It's also possible to call the [methodname]`executeJs()` method to access methods and fields of a Web Component using [methodname]`this.myFieldName`.

Reviewer: Is this a method call? If yes, then it should have the closing brackets `()`.

```suggestion
It's also possible to call the [methodname]`executeJs()` method to access methods and fields of a Web Component using `this.myFieldName`.
```

> If you do not need the element to be initialized before, you can use [methodname]`UI.getCurrent().getPage().executeJs` instead.

Reviewer: ```suggestion
If the element doesn't need to be initialized first, you can use [methodname]`UI.getCurrent().getPage().executeJs()` instead.
```
